### DISCUSSION TITLE
Changing User state without login

### DISCUSSION THREAD

**Discussion by DaveSH12** [dani-garcia/vaultwarden#4873] [Q&A] [answered]
Hey Community,
After a new user accepts the invitation and registers, their status remains 'invited' until they log in once. How can I change it so that the user only needs to create an account and the admin can confirm them without the user having to log in once?

Kind regards,
David

**Comment by BlackDex** [comment, marked as answer]
You can't

**Reply by DaveSH12** [reply]
ok, thank you
